### PR TITLE
Delete task

### PR DESCRIPTION
**Pull request by Yaladah**

### CODE REVIEW

**Comment by PawanPatil19** [comment]
Are you using any prefix for specifying the index in the **delete** command?

**Comment by Yaladah** [comment]
> Are you using any prefix for specifying the index in the **delete** command?

No, the changes to the format has been updated in the UG.

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -18,28 +18,28 @@ public class DeleteCommand extends Command {
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the person identified by the index number in the displayed task list or the person's name.\n"
```
Comment (zhengsterz) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java:21]: i think this sentence could be clearer as im not quite sure what it means

**Review by zhengsterz — COMMENTED** [review]
,LGTM

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java (hunk 1 above)
Comment (Yaladah) [reply src/main/java/seedu/address/logic/commands/DeleteCommand.java:21]: Oh I think this was a typo, thanks

**Pull request merged**